src/main/java/com/hermesworld/ais/galapagos/adminjobs/impl/NoUpdatesAdminClient.java

```diff
@@ -137,35 +137,35 @@ public DescribeConfigsResult describeConfigs(Collection<ConfigResource> resource
     @Override
     @Deprecated
     public AlterConfigsResult alterConfigs(Map<ConfigResource, Config> configs) {
-        throw new UnsupportedOperationException();
+        return delegate.alterConfigs(Map.of(), new AlterConfigsOptions().validateOnly(true));
```

Comment:
No need to pass empty configs here. Just pass `configs`, as it is still quite cool to have the server validate the set.